File: Application/SkuciSe/app/src/main/java/com/example/skucise/models/HelperModels/Filters.kt
package com.example.skucise.models

import com.example.skucise.models.HelperModels.FurnishedType
import com.example.skucise.models.HelperModels.HeatingType
import com.example.skucise.models.HelperModels.OldNewBuilding
import java.io.Serializable

class Filters(
    var city: String,
    var terrace: Boolean,
    var parking: Boolean,
    var wifi: Boolean,
    var tv: Boolean,
    var airCondition: Boolean,
    var kitchen: Boolean,
    var bathroom: Boolean,
    var alarm: Boolean,
    var pool: Boolean,
    var barbecue: Boolean,
    var firePlace: Boolean,
    var gym: Boolean,
    var rent: Boolean,
    var sell: Boolean,
    var house: Boolean,
    var flat: Boolean,
    var garage: Boolean,
    var parkingLott: Boolean,
    var businessArea: Boolean,
    var priceMin: Long,
    var priceMax: Long

    ) : Serializable{
    constructor() : this(
        "Kragujevac",
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        100,
        1000
        )
}

File: Application/Backend/SkuciSeAPI/SkuciSeAPI/Models/HelperModels/Filters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkuciSeAPI.Models.HelperModels
{
    public class Filters
    {
        public string City { get; set; }
        public Boolean Terrace { get; set; }
        public Boolean Parking { get; set; }
        public Boolean Wifi { get; set; }
        public Boolean Tv { get; set; }
        public Boolean AirCondition { get; set; }
        public Boolean Kitchen { get; set; }
        public Boolean Bathroom { get; set; }
        public Boolean Alarm { get; set; }
        public Boolean Pool { get; set; }
        public Boolean Barbecue { get; set; }
        public Boolean FirePlace { get; set; }
        public Boolean Gym { get; set; }
        public Boolean Rent { get; set; }
        public Boolean Sell { get; set; }
        public Boolean House { get; set; }
        public Boolean Flat { get; set; }
        public Boolean Garage { get; set; }
        public Boolean ParkingLott { get; set; }
        public Boolean BusinessArea { get; set; }
        public long PriceMin { get; set; }
        public long PriceMax { get; set; }
    }
}
